assets/locales/en.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Password_should_be_string": "The type of password should be a string",
+  "Password_should_combine_num_alpha": "The password should be a combination of numbers and alphabets",
+  "Password_length_invalid" : "The password should have a length between 6 and 64",
+  "Register_success" : "You have registered successfully. Please check for a validation mail sent to your mail within 3 days.",
```

Comment:
“Please check your inbox for a validation email to activate your account within three days.”